mimesniff/mime-types/resources/mime-types.json

```diff
@@ -113,6 +125,24 @@
     "navigable": true,
     "encoding": "GBK"
   },
+  {
+    "input": "text/html;charset=();charset=GBK",
+    "output": "text/html;charset=GBK",
```

Comment:
I get `text/html;charset="()"` for this. https://mimesniff.spec.whatwg.org/#http-quoted-string-token-code-point includes ( and ) so `()` is a fine charset.

Reply:
Yea, this we my mistake - was confused about the allowed characters.

Reply:
*was my mistake